instat/dlgInfillMissingValues.vb

```
@@ -306,7 +466,75 @@ Public Class dlgInfillMissingValues
         TestOkEnabled()
     End Sub
 
-    Private Sub ucrReceiverElement_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverElement.ControlContentsChanged, ucrSaveNewColumn.ControlContentsChanged, ucrPnlStartEnd.ControlContentsChanged, ucrInputConstant.ControlContentsChanged, ucrPnlMethods.ControlContentsChanged
+    Private Sub ucrPnlOptions_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlOptions.ControlValueChanged, ucrPnlMethods.ControlValueChanged, ucrInputComboType.ControlValueChanged
+        If rdoDisplay.Checked OrElse rdoShow.Checked Then
+            ucrBase.clsRsyntax.SetBaseRFunction(clsVisualizeElementNa)
+            ucrBase.clsRsyntax.iCallType = 3
+            Me.Size = New System.Drawing.Size(Me.Width, iDialogHeight * 0.8)
+            ucrBase.Location = New Point(ucrBase.Location.X, iBaseMaxY / 1.33)
+            ucrReceiverDisplayObserved.SetMeAsReceiver()
+            If rdoDisplay.Checked Then
+                Select Case ucrInputComboType.GetText
+                    Case "Distribution"
+                        ucrChkAddLegend.Visible = False
+                        clsVisualizeElementNa.AddParameter("type", Chr(34) & "distribution" & Chr(34), iPosition:=8)
+                    Case "Gap size"
+                        ucrChkAddLegend.Visible = True
+                        clsVisualizeElementNa.AddParameter("type", Chr(34) & "gapsize" & Chr(34), iPosition:=8)
+                    Case "Intervals"
+                        ucrChkAddLegend.Visible = True
+                        clsVisualizeElementNa.AddParameter("type", Chr(34) & "interval" & Chr(34), iPosition:=8)
+                End Select
```

should we have a `Case Else` to raise a developer error if `GetText` returns something unexpected?